style.css

```diff
@@ -174,15 +174,15 @@ header {
   font-family: 'Inter', sans-serif;
   padding: 10px 12px;
   margin: 12px 0 22px 0;
-  background: rgba(255, 255, 255, 0.24);
+  background-color: rgba(100, 100, 100, 1);
 }
 
 .textgrid:hover {
   border: 2px solid blue;
 }
 
 .seeproject {
-  background-color: #ff6b00;
+  background-color: #c15206;
```

Comment:
According to this [Button Contrast Checker](https://www.aditus.io/button-contrast-checker/), It looks like you still have some issues with the color contrast of your project. You can use the tool mentioned to check the issues and fix them as appropriate.